Cargo.toml

```diff
@@ -71,7 +71,7 @@ serde_json = "1.0.0"
 # Akamai example
 tokio = "=0.2.0-alpha.4"
 env_logger = { version = "0.5.3", default-features = false }
-rustls = "0.12"
-tokio-rustls = "0.5.0"
-webpki = "0.18"
-webpki-roots = "0.14"
+rustls = "0.16"
+tokio-rustls = { git = "https://github.com/quininer/tokio-rustls", branch = "tokio-0.2" }
```

Comment:
I believe there is a release of this on crates?

Reply:
@lucab - I think you're right.. it looks like the release is from that branch, in fact.